py4DSTEM/process/diffraction/digital_dark_field.py

```diff
@@ -494,47 +494,143 @@ def DDFimage(points_array, aperture_positions, Rshape=None, tol=1):
     return image
 
 
-def DDF_radial_image(points_array, radius, Rshape, tol=1):
+def radial_filtered_array(points_array_w_rphi, radius, tol=1):
```

Comment:
This method now returns a filtered array, and not a digital dark field image - can you update the first line of the docstring to reflect that?